File: c++/날짜(월,일)를 입력받아 1년 중 몇번째 날인지 계산.cpp
#include <iostream>

using namespace std;

int main() {
    int month, day;
    if (cin >> month >> day) {
        int days_in_month[] = {0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        int day_count = 0;
        for (int i = 1; i < month; ++i) {
            day_count += days_in_month[i];
        }
        day_count += day;
        cout << day_count << endl;
    }
    return 0;
}

File: java/날짜(월,일)를 입력받아 1년 중 몇번째 날인지 계산.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        if (scanner.hasNextInt()) {
            int month = scanner.nextInt();
            int day = scanner.nextInt();
            int[] daysInMonth = {0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
            int day_count = 0;
            for (int i = 1; i < month; i++) {
                day_count += daysInMonth[i];
            }
            day_count += day;
            System.out.println(day_count);
        }
    }
}
